README.md

```diff
@@ -35,6 +37,21 @@ differences:
   authenticating to the registry. Must be specified for private repos or when
   using `put`.
 
+* `ecr`: *Optional. Default `false`.* If set, Access Key ID and secret will be 
```

Comment:
Is this extra field necessary? Could this just be inferred when the other fields are set, or is there a time where *only* this field would be set? :thinking: